juniper-vpn.py

```diff
@@ -22,10 +22,10 @@
 import socket
 import netifaces
 import datetime
+import re
 
 debug = False
 
-ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
@russdill lol wow, why would anyone consider skipping ssl checks for software relating to vpn?